### ISSUE TITLE
[HOLD #13799] Chat gets unbold when opening the app again after closing it with the native home button

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#13684]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Navigate to a chat
2. Send a message
3. Long press on the message
4. Press mark as unread
5. Go to LHN and check the chat is bolded
6. Press native home button to close the app
7. Now open the app again
8. Check the chat again

## Expected Result:
The chat should stay bolded

## Actual Result:
The chat gets unbolded

## Workaround:
unknown

## Platform:

Where is this issue occurring?

- Android

**Version Number:** 1.2.41-0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/208424160-4d748e5c-5b4f-4fa8-a2bb-12a7b275028f.mp4

https://user-images.githubusercontent.com/43996225/208424198-25ee1e74-712d-48e4-87e7-4e3cdf569bfd.mp4

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @thesahindia 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1671279949227049

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by sonialiap** [comment]
I can reproduce behavior on my Android app following the outlined steps 👍

**Comment by mountiny** [comment]
Putting this on HOLD for the refactor of unread logic, it might be the case that we are breaking something during this change so lets have a look into these once the refactor is finished https://github.com/Expensify/Expensify/issues/227234

**Comment by roryabraham** [comment]
@mountiny I added testing steps for this issue into [this PR](https://github.com/Expensify/App/pull/13419), so we should hopefully be able to ensure that we fix it.

**Comment by mountiny** [comment]
Thank you!

**Comment by JmillsExpensify** [comment]
Adding myself to this issue as the BZ person that's going to help test all sequence number and related bugs when the appropriate PRs are merged.

**Comment by JmillsExpensify** [comment]
Still working through the intricacies of the sequence numbers project.

**Comment by puneetlath** [comment]
I believe this one actually should be fixed. @kavimuru could you re-test?

**Comment by JmillsExpensify** [comment]
Thanks for clarifying! I'll jump in and test now.

**Comment by JmillsExpensify** [comment]
Huh, @puneetlath I was actually able to still reproduce this on. Twice in fact. Notice how the chat in the LHN starts read (unbold), briefly goes unread (bold), and then settles on unread (unbold). If this part of the refactor is done, we should take this `internal` or open it up to contributors.

**Comment by JmillsExpensify** [comment]
Video is here. 

https://user-images.githubusercontent.com/1127863/211920649-3f1c9b5b-b615-4ca8-ae2c-b790ab50d483.mov

**Comment by puneetlath** [comment]
Ahhhh got it. I believe that is actually the same issue as this: https://github.com/Expensify/App/issues/13799. 

Basically, the way our navigator is implemented, when you open the app, we are rendering the LHN and your most recently visited chat (though it isn't visible). Which is why that message ultimately gets marked as read. So I think we should put it on hold for #13799.

**Comment by JmillsExpensify** [comment]
Ok cool, thanks for running me through that one. We'll hold in the meantime.

**Comment by puneetlath** [comment]
Still on hold.

**Comment by JmillsExpensify** [comment]
Adding this to the navigation project instead of sequence numbers given that the linked issue is also held on navigation.

**Comment by puneetlath** [comment]
That makes sense. I'm going to remove my assignment for now. I think just having one BZ assigned while it's on hold is enough. We'll probably be able to open this up to external once that project is done.

**Comment by JmillsExpensify** [comment]
Still held on navigation.

**Comment by JmillsExpensify** [comment]
Still held on navigation, though that design doc is now open for review. We're making progress!

**Comment by JmillsExpensify** [comment]
Still on hold.

**Comment by JmillsExpensify** [comment]
Still on hold.

**Comment by JmillsExpensify** [comment]
Still on hold.

**Comment by JmillsExpensify** [comment]
Still on Hold per the linked issue in title.

**Comment by JmillsExpensify** [comment]
Still on hold

**Comment by JmillsExpensify** [comment]
Still held on navigation.

**Comment by JmillsExpensify** [comment]
Still held.

**Comment by JmillsExpensify** [comment]
Going to start testing these types of issues this week. They're a lot, it'll take a bit.

**Comment by JmillsExpensify** [comment]
Still working through the testing issues.

**Comment by JmillsExpensify** [comment]
Same

**Comment by JmillsExpensify** [comment]
Aiming for end of July for this and all the other issues.

**Comment by JmillsExpensify** [comment]
Still on hold

**Comment by luacmartins** [comment]
on hold, not overdue

**Comment by JmillsExpensify** [comment]
Same

**Comment by JmillsExpensify** [comment]
Same

**Comment by JmillsExpensify** [comment]
I think monthly is the right priority. On HOLD.

**Comment by JmillsExpensify** [comment]
Same as last month.

**Comment by JmillsExpensify** [comment]
Same

**Comment by JmillsExpensify** [comment]
This is still as hold, as noted. That said, it would be interesting to have someone from @applausebot test this issue.

**Comment by JmillsExpensify** [comment]
Floated in #qa for another test run.

**Comment by JmillsExpensify** [comment]
This issue is no longer reproducible, so @thesahindia is due payment.

**Comment by JmillsExpensify** [comment]
Assigning to someone else for payment summary.

**Comment by trjExpensify** [comment]
Payment summary: 

$250 to @thesahindia for the bug report. 

Note: Price point is reflective of the time in which this bug was reported, prior to the change to $50.

**Comment by JmillsExpensify** [comment]
@thesahindia Can you request via NewDot? I'm going to close this issue and can circle back for approval.

**Comment by JmillsExpensify** [comment]
$250 approved for @thesahindia
